templates/common/infra/bicep/core/monitor/applicationinsights.bicep

```diff
@@ -4,6 +4,7 @@ param location string = resourceGroup().location
 param tags object = {}
 
 param logAnalyticsWorkspaceId string
+param useAPIM bool
```

Comment:
I'm not a fan of having to special case APIM throughout the files.

Reply:
please suggest a better name according to the naming convention

Reply:
While we don't figure out a way maybe it's better to just remove the specific dashboard for APIM. What do you think?

Reply:
What do we lose if we don't have it?  Is it required for APIM?

Reply:
it's not required. Just a way to explore the APIM metrics with the dashboard. I will remove it for now

Reply:
fixed